main.go

```diff
@@ -284,4 +292,7 @@ func initFlags(fs *pflag.FlagSet) {
 		":9440",
 		"The address the health endpoint binds to.",
 	)
+
+	logs.AddFlags(fs, logs.SkipLoggingConfigurationFlags())
```

Comment:
`logs.AddFlags(fs, logs.SkipLoggingConfigurationFlags())`

is not needed, it is later removed in the upstream version of this file.

Reply:
I was wrong, logsv1.addflags is at the top of the initFlags function in the latest main.go